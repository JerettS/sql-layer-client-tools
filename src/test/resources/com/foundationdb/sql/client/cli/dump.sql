--- c
---  o

DROP GROUP IF EXISTS c;
DROP TABLE IF EXISTS o;
DROP TABLE IF EXISTS c;

CREATE TABLE c(
  cid INT NOT NULL PRIMARY KEY,
  x INT
);

CREATE TABLE o(
  oid INT NOT NULL PRIMARY KEY,
  cid INT,
  GROUPING FOREIGN KEY(cid) REFERENCES c(cid),
  y INT
);

--- x

DROP TABLE IF EXISTS x;

CREATE TABLE x(
  xid INT
);
