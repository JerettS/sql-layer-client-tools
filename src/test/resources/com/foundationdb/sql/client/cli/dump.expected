WARNING: Cannot find the table `test`.`c`
ROWS: 0

WARNING: Cannot find the table `test`.`o`
ROWS: 0

WARNING: Cannot find the table `test`.`c`
ROWS: 0

ROWS: 0

ROWS: 0

WARNING: Cannot find the table `test`.`x`
ROWS: 0

ROWS: 0

